### PR TITLE
New setting to skip adventure map fadein/fadeout

### PR DESCRIPTION
**Pull request by dydzio0614**
As in title - allows skipping embark/ disembark/pickup animations etc.

### CODE REVIEW

File: Mods/vcmi/config/vcmi/english.json [hunk 1]
```diff
@@ -136,6 +136,8 @@
 	"vcmi.adventureOptions.leftButtonDrag.help" : "{Left Click Drag Map}\n\nWhen enabled, moving mouse with left button pressed will drag adventure map view.",
 	"vcmi.adventureOptions.smoothDragging.hover" : "Smooth Map Dragging",
 	"vcmi.adventureOptions.smoothDragging.help" : "{Smooth Map Dragging}\n\nWhen enabled, map dragging has a modern run out effect.",
+	"vcmi.adventureOptions.skipAdventureMapAnimations.hover" : "Skip delay-based animations",
+	"vcmi.adventureOptions.skipAdventureMapAnimations.help" : "{Skip delay-based animations}\n\nWhen enabled, Skips object fadeout and similar effects (resource collection, ship embark etc). Makes UI more reactive in some cases at the expense of aesthetics. Especially useful in PvP games. For maximum movement speed skipping is active regardless of this setting.",
```
Comment (IvanSavenko) [inline Mods/vcmi/config/vcmi/english.json:140]: Maybe simply 
Fade-out effects
Fade-out animations?

It is a bit more than that, true, but "delay-based animations" sounds weird...

Comment (dydzio0614) [reply Mods/vcmi/config/vcmi/english.json:140]: there are also fade-ins and possibly other cases that are not very fitting to be called "fading"

Comment (IvanSavenko) [reply Mods/vcmi/config/vcmi/english.json:140]: No other cases I think?
Fade-in and fade-out.
So fading effects / animations?

Comment (dydzio0614) [reply Mods/vcmi/config/vcmi/english.json]: done

**Pull request merged**